### ISSUE TITLE
Cannot assign to read only property 'crop' of object

### ISSUE THREAD

**Issue by guibulator** [sanity-io/sanity#4018]
**Describe the bug**

I was trying out the gallery image example: https://www.sanity.io/schemas/image-gallery-dea386ba and found a bug that prevents an image from being edited.

```
TypeError: Cannot assign to read only property 'crop' of object '#<Object>'
    at parseSource (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-XIWFTDF2.js?v=a84397f7:31401:23)
    at urlForImage (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-XIWFTDF2.js?v=a84397f7:31443:22)
    at ImageUrlBuilder2.url (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-XIWFTDF2.js?v=a84397f7:31802:16)
```

**To Reproduce**

1. [Go to my deployed studio](https://guibulator.sanity.studio/desk/post;b5689262-a2f3-4370-a14e-870e4f4169a3), I can grant you access or you can clone my github [repo](https://github.com/guibulator/sanity-getting-started/)
2. Click on the image gallery in the body component at the end of the page of a POST document
3. Edit and try to crop an image
4. Studio crashes with error-> Error: Cannot assign to read only property 'crop' of object '#<Object>'
5. Note The same image type is used in the Main image section and I am able to crop the image

**Expected behavior**
I should be able to crop the image

**Which versions of Sanity are you using?**

```
@sanity/eslint-config-studio  2.0.1 (up to date)
@sanity/vision                3.0.6 (latest: 3.1.2)
sanity                        3.1.2 (up to date)
```

**What operating system are you using?**
windows

**Which versions of Node.js / npm are you running?**
npm version 8.19.2
node version v18.12.1

**Additional context**

1. You can checkout the repo https://github.com/guibulator/sanity-getting-started/
2. The gallery is there https://github.com/guibulator/sanity-getting-started/blob/main/schemas/objects/gallery.ts
3. The gallery is  used in the bodyPortableText https://github.com/guibulator/sanity-getting-started/blob/main/schemas/objects/bodyPortableText.ts

**Comment by guibulator** [comment]
same as described in new issue #4026

**Comment by ninaandal** [comment]
Thank you for reporting @guibulator  🙌 A fix has been included in [v3.2.5](https://github.com/sanity-io/sanity/releases/tag/v3.2.5)

**Comment by guibulator** [comment]
No problem! I already tested with the latest version and it works well. Thanks again for the fix!
